workers/unity/Packages/io.improbable.gdk.querybasedinteresthelper/Constraint.cs

```diff
@@ -10,19 +10,19 @@ namespace Improbable.Gdk.QueryBasedInterest
     /// </summary>
     public class Constraint
```

Comment:
Do we want to add the `Self` constraint now it's supported?

Reply:
Probably, not in this PR though (this was actually introduced in 14.9.0)